data/human/news.txt

```diff
@@ -1990,6 +2093,75 @@ news "conspiracy theorist"
 
 
 
+news "players crew member"
+	location
+		government "Republic" "Free Worlds" "Neutral" "Independent" "Syndicate"
+	to show
+		"flagship crew" > 30
```

Comment:
Maybe a slightly larger number, so that the "never met" makes a bit more sense?

These are really funny.